isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -1,5 +1,5 @@
 #include "BundleAdjust.h"
-
+ ~
```

Comment:
Typo?

Reply:
weird, not on local. I'll see if pushing will fix it.